apps/spark/src/main/java/com/linkedin/openhouse/jobs/spark/DataCompactionSparkApp.java

```diff
@@ -58,6 +65,23 @@ protected void runInner(Operations ops) {
           fileGroupRewriteResult.rewrittenDataFilesCount(),
           fileGroupRewriteResult.rewrittenBytesCount());
     }
+    // Add compaction timestamp to table properties.
+    CompactionEvent event =
```

Comment:
@teamurko  can you confirm if this will be run after all partitions are compacted in Table or after  each partition is compacted in a Table?

Reply:
it will run after compaction is done on the whole table